### params for IBVS nodes ###
### Waypoints ###

p_des_mapper: {

p_des_outer: [-70, -70, 70, -70, 70, 70, -70, 70],

p_des_inner: [-70, -70, 70, -70, 70, 70, -70, 70]

}
